pilot/helpers/Debugger.py

```diff
@@ -75,11 +75,12 @@ def debug(self, convo, command=None, user_input=None, issue_description=None, is
                         result['completed_steps'] = steps[:step_index]
                         result['current_step'] = steps[step_index]
                         result['next_steps'] = steps[step_index + 1:]
+                        result['current_step_index'] = step_index
```

Comment:
step_index counter starts with 0 i think and in prompt we use 
`{% for step in completed_steps %}
step #{{ loop.index }}:`

which i belive starts counting at 1.

Reply:
oh, great catch. Will fix it